tools/mock-okta/tapes/a15f363b56ad798c9853a981332e6f0e.js

```diff
@@ -1,7 +1,7 @@
 var path = require("path");
 
 /**
- * GET /oauth2/default/v1/authorize?client_id=0oaom8GeLzZeX7oYZ0g3&redirect_uri=http://localhost:3000/authorization-code/callback&response_type=code&response_mode=query&state=STATE&nonce=NONCE&scope=openid email
+ * GET /oauth2/default/v1/authorize?scope=openid profile email&response_type=code&redirect_uri=http://localhost:8080/authorization-code/callback&state=STATE&client_id=0oap44gsvXUitBZSv0g3
```

Comment:
Remind me: Does ordering here matter? Do we rebuild this authorize request in mock-okta to ensure the hashes are the same?

Reply:
The authorize request is manually constructed here https://github.com/okta/samples-nodejs-express-4/blob/016cfb650ab743046f15793f32dad57489bcda6f/tools/mock-okta/index.js#L198

Reply:
Ordering of the query parameters don't matter. But the ordering of the values in the parameters matter.
E.g scope=openid profile will not match a tape recorded with scope=profile openid